src/styles/components/_stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
I think we should use all modificators for stars to change count of active stars

Reply:
Added a for loop to cover that